dashboard/package.json

```diff
@@ -27,6 +27,7 @@
     "d3-selection": "^3.0.0",
     "fabric": "^5.2.1",
     "framer-motion": "^6.5.1",
+    "gen-proto": "^0.0.1",
```

Comment:
Seems unrelated to the PR, please reset the changes

Reply:
> Seems unrelated to the PR, please reset the changes

fixed